kanata/deflayer/navigation_vim_hrm.kbd

```diff
@@ -0,0 +1,52 @@
+;; Vim-Navigation layer (optimized for home-row mods):
+;;  - right: Vim-like arrows on HJKL, home/end page up/down, mouse scroll
+;;  - left: Tab/Shift-Tab, prev/next, quit
+;;  - top: Super-num (i3/sway) or Alt-num (browser), zoom in/out
+
+;; The `lrld` action stands for "live reload". This will re-parse everything
+;; except for linux-dev, i.e. you cannot live reload and switch keyboard devices.
+
+(deflayer navigation
+  M-1  M-2  M-3  M-4  M-5  lrld M-6  M-7  M-8  M-9  M-0
+  @qut @pad bck  fwd  XX        home pgdn pgup end  @run
+  XX   S-tab tab tab  XX        lft  down up   rght @fun
+  XX   XX   XX   XX   XX    _   @mwl @mwd @mwu @mwr XX
+            del             _             esc
+)
+
+;; NumPad
+(deflayer numpad
+  _    _    _    _    _     _   _    _    _    _    _
+  XX   home up   end  pgup      @/   @7   @8   @9   XX
+  XX   lft  down rght pgdn      @-   @4   @5   @6   @0
+  XX   XX   XX   XX   XX    _   @,   @1   @2   @3   @.
+            @std           @nbs           @std
+)
+
+;; function keys
+(deflayer funpad
+  XX   XX   XX   XX   XX   XX   XX   XX   XX   XX   XX
+  f1   f2   f3   f4   XX        XX   XX   XX   XX   XX
+  f5   f6   f7   f8   XX        XX   XX   XX   XX   _
+  f9   f10  f11  f12  XX   XX   XX   XX   XX   XX   XX
+            _               _             _
+)
+
+(defalias
+  std (layer-switch base)
+  pad (layer-switch numpad)
+
+  fun (layer-while-held funpad)
+
+  ;; Replace XX by the keyboard shortcut to quit applications on your OS.
+  ;; qut C-q ;; [Command]-[Q]
+  qut XX ;; do nothing
```

Comment:
I’m convinced this action is relevant but I’m not sure it is easily discoverable, being hidden in this file… On the other hand, making it appear in the root config file (like the `@run` action) would be confusing because it’s only supported here.